pkg/loadbalancer/nfqlb/nfqlb.go

```diff
@@ -55,12 +55,12 @@ func New(name string, m int, n int, nfqueue int) (*nfqlb, error) {
 	return lb, nil
 }
 
-func (n *nfqlb) Activate(identifier int) error {
+func (n *nfqlb) Activate(index int, identifier int) error {
 	var stderr bytes.Buffer
 	cmd := exec.Command(
 		nfqlbCmd,
 		"activate",
-		fmt.Sprintf("--index=%d", identifier-1),
```

Comment:
Why not ditch this -1 thing while it's still time?

Reply:
Sorry, looks like this was already removed

Reply:
I had to re-add it, otherwise there will be an NBC. From the beginning we are generating identifier from 1 to 100 (and not 0 to 99), if we change it, then the old TAPA won't be compatible with this new PR.